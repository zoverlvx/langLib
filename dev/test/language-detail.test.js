import React from "react";
import LanguageDetail from "../js/containers/language-detail";
import {shallow} from "enzyme";

describe("<LanguageDetail />", () => {
	it("renders 1 <LanguageDetail /> component", () => { 
		const component = shallow(<LanguageDetail />);
		expect(component).toHaveLength(1);
	});
});

