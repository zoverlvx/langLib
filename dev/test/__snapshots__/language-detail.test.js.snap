// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LanguageDetail /> provides a snapshot of LanguageDetail's tree 1`] = `
<Route
  render={[Function]}
/>
`;
